# Generated by Django 4.2.6 on 2024-05-02 18:13

from django.db import migrations, models
import django.db.models.deletion
import shopapp.libraries.validations


class Migration(migrations.Migration):

    dependencies = [
        ('shopapp', '0035_user_phone_number_alter_user_date_joined_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='profile',
            name='image',
            field=models.ImageField(default='/media/profile_pics/default.jpeg', help_text='Get a representative profile image.', upload_to='profile_pics'),
        ),
        migrations.AlterField(
            model_name='review',
            name='created_by',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='reviews', to='shopapp.profile'),
        ),
        migrations.AlterField(
            model_name='user',
            name='first_name',
            field=models.CharField(error_messages={'unique': 'First name is invalid. Try again.'}, max_length=50, validators=[shopapp.libraries.validations.FirstNameValidator()]),
        ),
        migrations.AlterField(
            model_name='user',
            name='last_name',
            field=models.CharField(error_messages={'unique': 'Last name is invalid. Try again.'}, max_length=50, validators=[shopapp.libraries.validations.LastNameValidator()]),
        ),
        migrations.AlterField(
            model_name='user',
            name='phone_number',
            field=models.CharField(blank=True, help_text='Format: 07********', max_length=10),
        ),
    ]
